content/download/debian.adoc

```diff
@@ -6,63 +6,194 @@ weight = 10
 :icons: fonts
 :iconsdir: /img/icons/
 
-== Debian Stable (Stretch)
+== Basic Information
 
-{{< repology debian_stable_backports >}}
+The information further down about the available KiCad packages might not
+always be fully up2date and correct due ongoing development. For getting an
+overview about the current most recent version for the KiCad packages within
+the various Debian releases please have a look at
+https://packages.debian.org/search?lang=en&keywords=kicad[package.debian.org].
 
-{{< repology debian_stable >}}
+Please note also that Debian stable releases don't get updates for existing
+packages except security updates. Packages within the Stable releases get
+usually newer and recent versions through backports repositories. KiCad is
+here no special case.
 
-The release 4.0.5 is available for Debian
-https://packages.debian.org/stretch/kicad[stable/stretch], the backported
-version 5.0.1 is available in
-https://packages.debian.org/stretch-backports/kicad[stretch-backports].
+To be able to install versions from a official Debian Backport repository you
+need to extend your sources.list configuration if not yet happen. If you are
```

Comment:
You may want to make sources.list "code style" as well

Reply:
Agreed. Changed within next force-push.